src/components/logical-expression-builder/input-components/text-input/text-input.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="form-group m-b-nil" [ngClass]="{ 'has-error': !_valid }">
+    <input
+        type="text"
+        class="form-control"
+        [value]="value"
```

Comment:
Can we not use `ngModel` here? Instead of manually updating the value ourselves?